common/src/locales/vi.json

```
@@ -480,7 +480,39 @@
     "copyBookingLink": "Sao chép liên kết đặt lịch",
     "createAppointment": "Tạo lịch hẹn",
```

Biome (2.5.3)

[error] The key createAppointment was already declared.

(lint/suspicious/noDuplicateObjectKeys)